changelog/fragments/bundle-validate-optional-validators.yaml

```diff
@@ -0,0 +1,11 @@
+entries:
+  - description: >
+      Added the `--select-optional` flag to `bundle validate` to pass a label selector
+      that runs any matching optional validator on top of required validators,
+      and `--list-optional` to display optional validators, their selectors, and
+      a short explanation of what each does.
+    kind: addition
```

Comment:
@estroz, 

Scorecard has a similar feature. Should not better we use the same names? 
--list-selector an --selector? 

Since we cannot remove/change the flags I think it is very important we be very conscious over its names. 
/hold

Reply:
There are always required validators being run, so you're only able to select optional validators, therefore `--{list-}selector` would be disingenuous and `--{list,select}-optional` makes sense.

Reply:
OK. Far enough.